data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -399,18 +399,25 @@ popup-separator-menu-item {
 
 .app-well-app {
     .overview-icon { @extend %app-well-icon; }
-    & > .overview-icon.overview-icon-with-label {
-        /* Use defaults here to "remove" the values set in _common.scss */
-        padding: 0px;
+    .overview-icon.overview-icon-with-label {
+        padding: 6px;
         spacing: 6px;
-    }
-}
+        > StBoxLayout {
```

Comment:
Isn't it missing a `&` here?

Reply:
Copied from https://github.com/endlessm/gnome-shell/blob/master/data/theme/gnome-shell-sass/widgets/_app-grid.scss#L29, tested and it works fine.